mining.md

```diff
@@ -250,7 +250,7 @@ An eligible miner can start by filling out `Parents`, `Tickets` and `ElectionPro
 
 Next, they compute the aggregate state of their selected parent blocks, the `ParentState`. This is done by taking the aggregate parent state of the blocks' parent TipSet, sorting the parent blocks by their tickets, and applying each message in each block to that state. Any message whose nonce is already used (duplicate message) in an earlier block should be skipped (application of this message should fail anyway). Note that re-applied messages may result in different receipts than they produced in their original blocks, an open question is how to represent the receipt trie of this tipsets 'virtual block'. For more details on message execution and state transitions, see the [Filecoin state machine](state-machine.md) document.
 
-Once the miner has the aggregate `ParentState`, they must apply the mining reward. This is done by adding the correct amount to the miner owner's account balance in the state tree. (TODO: link to block reward calculation. Currently, the block reward is a fixed 1000 filecoin).
+Once the miner has the aggregate `ParentState`, they must apply the mining reward. This is done by adding the correct amount to the miner owner's account balance in the state tree. See [block reward](#block-rewards) for details.
```

Comment:
In go-filecoin we currently transfer the mining reward before applying the state transformations from the messages. This replicates the old behavior where we sorted the reward message first. It's easy either way, but we either need to change it in the code or change it here.

Reply:
I think it largely doesnt matter all that much, but since ethereum applies it at the end of the block processing, i think it would be best to keep things the same as they do it.